### PR TITLE
Add a BBE for SFTP listener and other naming related changes

### PR DESCRIPTION
**Pull request by Maninda**
## Purpose

Resolves https://github.com/ballerina-platform/ballerina-standard-library/issues/1985
Resolves https://github.com/ballerina-platform/ballerina-standard-library/issues/1581

### CODE REVIEW

File: examples/index.json [hunk 1]
```diff
@@ -2125,8 +2125,8 @@
                 "isLearnByExample": false
             },
             {
-                "name": "Receive Emails Using a Listener",
-                "url": "receive-email-using-listener",
+                "name": "Listener",
+                "url": "ftp-listener",
```
Comment (ldclakmal) [inline examples/index.json:2129]: This is related to email listener. Right? Seems like the email files are not renamed.

Comment (Maninda) [reply examples/index.json]: Reverted the change.

File: examples/sftp-client/sftp_client.out [hunk 2]
```diff
@@ -0,0 +1,4 @@
+bal run sftp.bal
```
Comment (ldclakmal) [inline examples/sftp-client/sftp_client.out:1]:
```suggestion
bal run sftp_client.bal
```

File: examples/sftp-listener/sftp_listener.out [hunk 3]
```diff
@@ -0,0 +1,4 @@
+bal run sftp.bal
```
Comment (ldclakmal) [inline examples/sftp-listener/sftp_listener.out:1]:
```suggestion
bal run sftp_listener.bal
```

File: examples/sftp-client/sftp_client.out (hunk 2 above)
Comment (Maninda) [reply examples/sftp-client/sftp_client.out]: Changed all the output content using the file names given in the code itself.

File: examples/sftp-listener/sftp_listener.out (hunk 3 above)
Comment (Maninda) [reply examples/sftp-listener/sftp_listener.out]: Changed all the output content using the file names given in the code itself.

**Review by ldclakmal — APPROVED** [review]

**Pull request merged**